File: Paradigmas de linguagem de programação/Lista de Exercicio - PLP/Questão 06/Python/main.py
class Motor:
    def __init__(self, potencia):
        self.potencia = potencia

class Carro:
    def __init__(self, motor):
        self.motor = motor

motor = Motor(310)
carro = Carro(motor)
print("Potência do motor:", carro.motor.potencia)

File: Paradigmas de linguagem de programação/Lista de Exercicio - PLP/Questão 06/Go/main.go
package main
import "fmt"

type Motor struct {
    Potencia int
}

type Carro struct {
    Motor Motor
}

func main() {
    motor := Motor{Potencia: 310}
    carro := Carro{Motor: motor}
    fmt.Println("Potência do motor:", carro.Motor.Potencia)
}
